cocos/2d/renderer/render-data.ts

```diff
@@ -26,10 +26,11 @@ import { DEBUG, JSB } from 'internal:constants';
 import { assert } from '@base/debug';
 import { assertIsTrue } from '@base/debug/internal';
 import { memop } from '@base/utils';
+import { Color } from '@base/math';
```

Comment:
Why Color belongs to math module?

Reply:
for our current implementation, it does belong to math module, and it's our public interface design

Reply:
we could migrate `Color` to `core` layer, but it should follow the interface deprecation workflow, to deprecate the public interface `math.Color`